src/GraphQLClient.php

```diff
@@ -25,6 +29,16 @@ abstract class GraphQLClient
      */
     protected $options = [];
 
+    /**
+     * total timeout of the request in seconds
```

Comment:
types ?

Reply:
its float , I will mention this in doc block